apps/services/handlers.py

```diff
@@ -116,15 +176,39 @@ async def cancel_request(university_id: int, user_request_id: int, cancel_reques
             "user_request_id": user_request_id,
             "status_id": cancel_request.status_id
         },
-        "message": f"Canceled request with id {user_request_id}"
+        "message": f"Canceled request with id {user_request_id}",
+        "code": http_status.HTTP_202_ACCEPTED
```

Comment:
im not sure that 202 is correct

Reply:
Couldn't find 100 % appropriate, but google says its ok. ?

Reply:
For a PUT request: HTTP 200, HTTP 204 should imply "resource updated successfully". HTTP 201 if the PUT request created a new resource.

so - 200 is ok for this example